scripts/upgrade-v9.mjs

```diff
@@ -300,7 +300,7 @@ async function processBoxComponents(components, shouldReplace = false) {
       let valuesNotTransformed = []
 
       // Add default flex class for Flex components
-      if (component.componentType === 'Flex') {
+      if (component.componentType === 'Flex' || component.componentType === 'Stack') {
```

Comment:
What about `<Stack />` without `direction="row"` ? This will add a `flex` class which will lay out children horizontally rather than vertically.

Reply:
https://github.com/WTTJ/welcome-ui/pull/2902